update design contribution page (#10361)
ethereum/ethereum-org-website: src/content/contributing/design/index.md

### <Emoji text=":six:" size={1} /> Write design-related content on the website {#write-design-articles}

> The Ethereum developer community is strong, but the design community is falling slightly behind. If you are a designer with web3 knowledge, please consider sharing your learnings with the larger community so that we can all grow and improve together! We have a [design section](/developers/docs/design-and-ux/) that needs more articles!

Reviewer: ```suggestion
The Ethereum developer community is strong, but the design community is falling slightly behind. If you are a designer with web3 knowledge, please consider sharing your learnings with the larger community so that we can all grow and improve together, we have a [design section](/developers/docs/design-and-ux/) that needs more articles!
```

> The Ethereum developer community is strong, but the design community is falling slightly behind. If you are a designer with web3 knowledge, please consider sharing your learnings with the larger community so that we can all grow and improve together, we have a [design section](/developers/docs/design-and-ux/) that needs more articles!

Reviewer 2: ```suggestion
The Ethereum developer community is strong, but the design community is falling slightly behind. If you are a designer with web3 knowledge, please consider sharing your learnings with the larger community so that we can all grow and improve together; we have [a page on designing for Ethereum](/developers/docs/design-and-ux/) you can contribute to.
```
